src/components/inputs/Dates/DateRange/DateRange.types.ts

```diff
@@ -45,6 +45,11 @@ export type Props = IrisInputProps<{
    * @presetValue is the value of preset which is clicked.
    */
   onPresetClick?: (presetValue: PresetValue) => void;
+  /**
+   * The value used for the intialState of the range and the draft of the DateRange.
+   *
+   */
+  defaultValue?: [Date | null, Date | null];
```

Comment:
TODO: update this value so it is correct

Reply:
This actually looks right to me.